globalid.gemspec

```diff
@@ -16,7 +16,10 @@ Gem::Specification.new do |s|
   s.files        = Dir['MIT-LICENSE', 'README.md', 'lib/**/*']
   s.require_path = 'lib'
 
-  s.add_runtime_dependency 'activesupport', '>= 4.2.0'
+  s.add_runtime_dependency 'activesupport', ['>= 3.2.0', '< 4.0']
```

Comment:
This Rails version constraint satisfies our present needs, but will obviously need to be updated when we update saba to Rails 4.  I've tried running this code with Rails 4, and get an ambiguous test failure (something about railties again).  If people think it's worth dealing with now, we can _probably_ get this working up through Rails 4.1 or 4.2

Reply:
On Rails 4, can we just switch back to using the upstream version of the gem?

Reply:
@brenthoughton2, only if we jump from Rails 3.2.22 to Rails 4.2.  But fortunately, I think I've solved this issue.